workflow/snakemake_rules/nextclade.smk

```diff
@@ -182,6 +190,6 @@ rule generate_metadata:
         """
         ./bin/join-metadata-and-clades \
             {input.existing_metadata} \
-            {input.new_metadata} \
+            {input.nextclade_output[0]} \
```

Comment:
Kinda awkward and unclear why this accesses the first item in the list, i.e. you have to go tracing what that item is.

Would be nicer to be able to reference `input.new_metadata` instead, which should be possible with the suggestion above.

Reply:
(See [comment](https://github.com/nextstrain/ncov-ingest/pull/320#discussion_r933554543) above)